packages/mds-db/migration.ts

```diff
@@ -1,37 +1,24 @@
 import { csv } from '@mds-core/mds-utils'
 
 import logger from '@mds-core/mds-logger'
-import schema, { COLUMN_NAME, TABLE_NAME } from './schema'
+import { GeographyRepository } from '@mds-core/mds-geography-service'
+import { PolicyRepository } from '@mds-core/mds-policy-service'
+import schema, { COLUMN_NAME, MANAGED_TABLE_NAME } from './schema'
 import { SqlExecuter, MDSPostgresClient } from './sql-utils'
 
 // drop tables from a list of table names
 async function dropTables(client: MDSPostgresClient) {
   const exec = SqlExecuter(client)
-  const drop = csv(schema.DEPRECATED_TABLES.concat(schema.TABLES))
+  const drop = csv(schema.DEPRECATED_TABLES.concat(schema.MANAGED_TABLES))
   await exec(`DROP TABLE IF EXISTS ${drop};`)
+  await Promise.all([GeographyRepository.revertAllMigrations(), PolicyRepository.revertAllMigrations()])
```

Comment:
If we're already dropping these tables, won't the migration reverts barf?

Reply:
They aren't being dropped. They are excluded from the `DROP TABLE` command.